app/src/main/java/org/fossasia/phimpme/accounts/AccountActivity.java

```diff
@@ -242,7 +238,7 @@ public void onClick(DialogInterface dialog, int which) {
     }
 
     private void signInDropbox() {
-        if (accountPresenter.checkAlreadyExist(accountName[7]))
+        if (accountPresenter.checkAlreadyExist(accountName[8]))
```

Comment:
@the-dagger Yes using constants is a better idea. 
I think even more better approach is if we use Enumerated Type in the model class itself, that helps us to avoid using indexing of an array all the time, which is highly error prone.